src/workerd/jsg/modules-new.h

```diff
@@ -777,6 +789,20 @@ class ModuleRegistry final: public kj::AtomicRefcounted, public ModuleRegistryBa
   friend class Module::Evaluator;
 };
 
+// Dynamic import callback functions
+v8::MaybeLocal<v8::Promise> dynamicImport(v8::Local<v8::Context> context,
+    v8::Local<v8::Data> host_defined_options,
+    v8::Local<v8::Value> resource_name,
+    v8::Local<v8::String> specifier,
+    v8::Local<v8::FixedArray> import_attributes);
+
+v8::MaybeLocal<v8::Promise> dynamicImportWithPhase(v8::Local<v8::Context> context,
+    v8::Local<v8::Data> host_defined_options,
+    v8::Local<v8::Value> resource_name,
+    v8::Local<v8::String> specifier,
+    v8::Local<v8::FixedArray> import_attributes,
+    v8::ModuleImportPhase phase);
```

Comment:
Do these need to be added to the header? I'd prefer to keep these defined only within modules-new.c++ if possible.